sec1_intro_and_setup/test_app/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+Dh5MVXfUMV0OP/e9/frLAeNwrA5iUSi/pO3BDe6/L9l3c0mRUCCQr7XZyodVJ0V0HpmNbjFbmFOCIPx/t1lbF9Gvuf3pLEc4FvOTT6XcbgJolJdyMlbCJKK3T2dbdZoU1WuKnmP7CMlZ6so+mQroMSEhSDQd9Ww9nPfILTM1WHH14iJ3ZP4rgGAN8zCeT7+BNr/U7UmgEXf+tbV+hT7mW6UX2fnnygHeeX+ziAMcjCuI+XdT1RHGnvqUDYhh2rHqr7n3/taD3H2w4f6WN7hsHRO9M7CUBxO3w4VYUtWrudvKMWCLBhyoES/YsWW0ZtN96LfTyMqlVlkYN7r60GHuVjmdoINfZl0U66GmAuQfiZtkSLoJOu2o0db9C1+rgp+39tr2zyTmwIEixMDXspf2Jh4jusihyMqMUqct--AuI4K6kWzViwj+hK--w4t5EjeQoBTdZGmigYrZUQ==
```

Comment:
Whats this file for?

Reply:
`config/credentials.yml.enc` is an encrypted file which stores the credentials and is only decrypted by the `config/master.key`.

Reply:
From what you are saying, that means if both of us were to work on the same project, for both of our local to be able to decrypt this, we both need the master.key right?

If thats the case, why isn't the master.key commtited to git?

Reply:
`master.key` is not committed to git to due to security concerns. Although `credentials.yml.enc` is encrypted, it does not mean the file is non-breakable if the file gets to the wrong hands.

Reply:
ok. Then how can I decrypt the data to be able to run your code on my computer?

Reply:
One can provide Rails with your master key either using environmental specific credentials or global credentials (if environmental specific credentials is not present, Rails will choose global credentials by default)

In environmental specific credentials, `RAILS_MASTER_KEY` ENV variable is created. Rails will detect it and use it as your master key.
In global credentials, `config/master.key` is placed in a server where you can share it with the team.

Reply:
can you please explain everything using your own words instead of copy paste from somewhere?
I think this is the 2nd time i'm telling you this already.

Reply:
Either share the master key to you using communication channel (slack, email etc) or use the `RAILS_MASTER_KEY` environment variable